service/keypairmanager.go

```diff
@@ -63,6 +69,16 @@ func (db *DB) CreateKeypairTable() error {
 	return err
 }
 
+// AlterKeypairTable adds extra fields to an existing keypair database table
+func (db *DB) AlterKeypairTable() error {
+	_, err := db.Exec(alterKeypairAddAssertion)
+	if err != nil {
+		// Ignore error as the field may already be added
+		return nil
+	}
+	return err
```

Comment:
The "Alter" methods are the ones that most likely will get additions in the future, so I used that structure. Probably unnecessary, as you say.